gunfolds/solvers/clingo_rasl.py

```diff
@@ -273,7 +276,10 @@ def drasl_command(g_list, max_urate=0, weighted=False, scc=False, dm=None, bdm=N
     n = len(g_list)
     command = clingo_preamble(g_list[0])
     if scc:
-        command += encode_list_sccs(g_list)
+        if weighted and scc_members:
```

Comment:
I suggest removing the `if then else` entirely and only leaving one line 
```suggestion
        command += encode_list_sccs(g_list, scc_members)
```
This gives us a chance to specify SCC composition even if we are not doing optimization. Useful for figuring out how much does wrong SCC encoding affect the results.